### PR TITLE
Add graphic showcasing evolution of life

### PR DESCRIPTION
**Pull request by hendricius**
After reading a really good book by Ed Young on microorganism I figured I update the history of sourdough bread a bit more.

image

### CODE REVIEW

**Comment by cedounet** [comment]
I am not sure i understand the scaling on your timeline?  you normalize earth age to 1 year?  IMO it would be clearer if you put 20M years ago etc...  

I promise I will not be asking for a log scale 🤓

File: book/history/sourdough-history.tex [hunk 1]
```diff
@@ -9,11 +9,61 @@ \chapter{The history of sourdough}%
     lessons from the past.
 \end{quoting}
 
-Sourdough has been made since ancient times. The exact origins of fermented
+The story of sourdough bread begins in ancient oceans. These oceans were the
+birthplace of all Earth's life. To understand this timeline, envision Earth's
+history compressed into one year. On this scale, January 1st signifies Earth's
```
Comment (cedounet) [inline book/history/sourdough-history.tex:14]: Not sure exactly how you do that but shall be upper script.

also insécable space between month and day.

File: book/history/sourdough-history.tex [hunk 2]
```diff
@@ -9,11 +9,61 @@ \chapter{The history of sourdough}%
...
+\begin{figure}[!htb]
+\begin{center}
+  \input{figures/fig-life-planet-sourdough-timeline.tex}
+  \caption[Sourdough microbiology timeline]{Timeline of significant events
+    starting from the first day of Earth's existence,
+    divided into months, and extending to the present day,
+    marked at midnight. This visualization shows the pivotal steps
+    of life and sourdough on earth.}%
+\end{center}
+\end{figure}
```
Comment (cedounet) [inline book/history/sourdough-history.tex:33]: Not sure this is clearer than showing millions of years… I see why you are trying to say but I need to think about it.

File: book/history/sourdough-history.tex [hunk 3]
```diff
@@ -9,11 +9,61 @@ \chapter{The history of sourdough}%
...
+
+The first arrivals, bacteria or archaea, remains debated. For three
```
Comment (cedounet) [inline book/history/sourdough-history.tex:35]: What do you mean by debated ? Who arrived first ?

File: book/history/sourdough-history.tex [hunk 4]
```diff
@@ -9,11 +9,61 @@ \chapter{The history of sourdough}%
...
+first aquatic fungi appeared. They adapted and, by November 24th, had colonized
+land.
+
+By December 3rd, yeasts emerged on land. This laid groundwork for bread-making.
+Jump 140 million years to December 14th, and dinosaurs arose. They reigned
+until December 29th when they faced extinction. Another 25 million
+years later, or our timeline's 2 days, humans appeared.
+
+A few hours later after the arrival of humans, a more subtle culinary
+revolution was unfolding. By 12,000 BC, just 5 seconds before our metaphorical
```
Comment (cedounet) [inline book/history/sourdough-history.tex:53]: Use Siunit package here there is an example further down the text \num{12000}~BC

File: book/history/sourdough-history.tex [hunk 5]
```diff
@@ -9,11 +9,61 @@ \chapter{The history of sourdough}%
...
+midnight, the first sourdough breads were being baked in ancient Jordan. A blink of
+an eye later, or 4 seconds in our time compression, Pasteur's groundbreaking work
+with yeasts set the stage for modern bread-making. From the moment this book
+began to take shape to your current reading, only milliseconds have ticked by.
+~\cite{Yong_2017}
```
Comment (cedounet) [inline book/history/sourdough-history.tex:58]: Put final dot after the citation.

**Review by cedounet — CHANGES_REQUESTED** [review]

File: book/history/sourdough-history.tex (hunk 2 above)
Comment (hendricius) [reply book/history/sourdough-history.tex:33]: I got the idea from Ed Young's book "I contain multitudes". I kind of like it as it shows how recent all human innovation has been. Also shows how lonely the planet has been for some time. My plan is to add another one with 12000 BC till now once this PR is merged 😅

**Comment by hendricius** [comment]
@cedounet thanks for the Feedback! I fixed most of the issues 👍 . As a basis to calculate I used: "(4.54 billion years / 12 months = 0.3785 billion years/month". I also added it as a comment in the code 😎

File: book/history/sourdough-history.tex (hunk 2 above)
Comment (cedounet) [reply book/history/sourdough-history.tex:33]: Have you tried to also have non scaled dates maybe in a different colour?

I would also add some key dates like Pangea (you already have dinosaurs extinction).  Just an idea.

**Comment by cedounet** [comment]
You might want to reword your dates as well to match American style https://editorsmanual.com/articles/dates-american-vs-british/

this is pretty much what I have in the Chicago manual of style… not ordinals.

if we ever want to translate then we should use datetime2 package.  I would use it anyway as I would get the date format wrong without looking on the inter webs.

File: book/history/sourdough-history.tex (hunk 2 above)
Comment (hendricius) [reply book/history/sourdough-history.tex:33]: Pangea is a funny idea 👍 . Just not sure I can fit more into that one chart 😅

Comment (hendricius) [reply book/history/sourdough-history.tex:33]: Also - TLDR - it was that recent the continents formed 😅

**Comment by hendricius** [comment]
> You might want to reword your dates as well to match American style https://editorsmanual.com/articles/dates-american-vs-british/
> 
> this is pretty much what I have in the Chicago manual of style… not ordinals.
> 
> if we ever want to translate then we should use datetime2 package. I would use it anyway as I would get the date format wrong without looking on the inter webs.

Sweet! Thanks for the tip. I updated - hope I didn't miss something. `datetime2` sounds like a good idea too. Let me see if I can add this with the next PR for the sourdough human timeline.

File: book/history/sourdough-history.tex [hunk 6]
```diff
@@ -28,7 +78,7 @@ \chapter{The history of sourdough}%
 increased in size and smelled funky. She decided to bake
 the dough anyway and was rewarded with a much
 lighter, softer, better tasting bread dough. From that day
-on she continued to make bread this way.
+on she continued to make bread this way.~\cite{egyptian+bread}
```
Comment (cedounet) [inline book/history/sourdough-history.tex:81]: Punctuation vs quote

**Comment by hendricius** [comment]
Thanks! I fixed the issues. Also added pangea and squeezed it in. Awesome idea. Can you check one more time please? Thank you!

File: book/history/sourdough-history.tex [hunk 7]
```diff
@@ -9,11 +9,66 @@ \chapter{The history of sourdough}%
...
+Now delving deeper into the realm of sourdough, it can likely be traced to aforementioned
+Ancient Jordan~\cite{jordan+bread}. Looking at the earth's timeline sourdough
+bread can be considered a very recent invention.
+
+The exact origins of fermented
 bread are, however, unknown. One of the most ancient preserved
-sourdough breads has been excavated in Switzerland.
-However, based on recent research, some scientists speculate that sourdough
-bread had already been made in \num{12000}~BC in ancient Jordan~\cite{jordan+bread}.
+sourdough breads has been excavated in Switzerland.~\cite{switzerland+bread}
```
Comment (cedounet) [inline book/history/sourdough-history.tex:71]: Citation before the dot

**Review by cedounet — COMMENTED** [review]
Maybe run make check on it.  I could only spot one citation mistake.

text ok time chart could be slanted for better readability maybe?

I would add the scaling bottom left of the graph… 1 quarter represents 1 billion year about.

**Comment by hendricius** [comment]
Great idea 👍. Also thanks for the hint on make check.

**Review by cedounet — APPROVED** [review]

**Comment by cedounet** [comment]
I really like it now.  Have you tried to use colours?

**Comment by hendricius** [comment]
Great idea 🙏🏻

**Comment by hendricius** [comment]
@cedounet what do you think? I also didn't want to overdo the colors.

image

**Comment by cedounet** [comment]
Yeah it's a matter of balance... LGTM, we can always fine-tune later anyway.

**Pull request merged**